packages/mask/src/components/InjectedComponents/DisabledPluginSuggestion.tsx

```diff
@@ -52,25 +54,63 @@ export function PossiblePluginSuggestionPostInspector() {
     return <PossiblePluginSuggestionUI plugins={matches} />
 }
 export function PossiblePluginSuggestionUI(props: { plugins: Plugin.DeferredDefinition[] }) {
-    const { t } = useI18N()
-    const t2 = usePluginI18NField()
     const { plugins } = props
+    const onClick = useCallback((x: Plugin.DeferredDefinition) => {
+        Services.Settings.setPluginMinimalModeEnabled(x.ID, false)
+    }, [])
     if (!plugins.length) return null
+
     return (
         <>
             {plugins.map((x) => (
                 <MaskPostExtraInfoWrapper
                     open
                     key={x.ID}
-                    title={t('plugin_not_enabled', { plugin: t2(x.ID, x.name) })}
+                    title={<PluginI18NFieldRender field={x.name} pluginID={x.ID} />}
+                    publisher={
+                        x.publisher ? <PluginI18NFieldRender pluginID={x.ID} field={x.publisher.name} /> : undefined
+                    }
+                    publisherLink={x.publisher?.link}
                     action={
-                        <Switch
-                            sx={{ marginRight: '-12px' }}
-                            onChange={() => Services.Settings.setPluginMinimalModeEnabled(x.ID, false)}
-                        />
+                        <Button
+                            size="small"
+                            startIcon={<PluginIcon />}
+                            variant="contained"
+                            onClick={() => onClick(x)}
+                            sx={{
+                                backgroundColor: MaskColorVar.buttonPluginBackground,
+                                color: 'white',
+                                width: '254px',
+                                '&:hover': {
+                                    backgroundColor: MaskColorVar.buttonPluginBackground,
+                                },
+                            }}>
+                            Enable plugins
+                        </Button>
                     }
+                    content={<Rectangle />}
                 />
             ))}
         </>
     )
 }
+
+const useRectangleStyles = makeStyles()(() => ({
+    rectangle: {
+        height: 8,
+        background: 'rgba(255, 255, 255, 0.5)',
+        borderRadius: 8,
+    },
+}))
+interface RectangleProps extends HTMLProps<HTMLDivElement> {}
+
+export function Rectangle(props: RectangleProps) {
+    const { classes } = useRectangleStyles()
+    return (
+        <div {...props}>
+            <div className={classes.rectangle} style={{ width: 103 }} />
```

Comment:
Try this https://mui.com/material-ui/react-skeleton/.

Reply:
@lelenei